deploy/helm/templates/asserts.yaml

```diff
@@ -16,4 +16,8 @@
 
 {{- if eq ((semver "3.9.0").Compare (semver .Capabilities.HelmVersion.Version)) 1 -}}
 {{ fail "This version of Helm is not supported. Please use 3.9.0 or newer." }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{- if and (or (not (empty .Values.cluster.filter.exclude_namespaces)) (not (empty .Values.cluster.filter.exclude_namespaces_regex))) (or (not (empty .Values.cluster.filter.include_namespaces)) (not (empty .Values.cluster.filter.include_namespaces_regex))) -}}
```

Comment:
Is this the same as `isNamespacesFilterEnabled` macro? can you use it here?

Reply:
it is not the same